openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -137,6 +138,19 @@ class Meta:
         abstract = True
 
 
+class CourseSequenceSpecialExamMixin(models.Model):
+    """
+    This mixin stores XBlock information that affects outline level information
+    pertaining to special exams
+    """
+    is_practice_exam = models.BooleanField(default=False)
+    is_proctored_enabled = models.BooleanField(default=False)
+    is_timed_exam = models.BooleanField(default=False)
```

Comment:
@symbolist is doing the real review, but a quick comment: I like how you separated the `ExamData` from the rest of the attrs structs because logically speaking (regardless of what's in the XBlock), not all sequences are exams. The `CourseContentVisiblityMixin` exists because those fields have to go into two separate models. But exams are logically distinct enough that I think it would be useful to model them separately in their own table that has a unique foreign key to `CourseSectionSequence`.

Reply:
@pcockwell As requested can you create a `CourseSequenceExam` model with these 3 fields and a `OneToOneField` to CourseSectionSequence?